### PR TITLE
[RedSuns Review] Tutorial Maintenance(Keras)

### PR DESCRIPTION
**Pull request by ueno-hiroshi002**
## Pull Request Description:
* Modify old tutorials
  * Add install versions at below libraries.
    * tensorflow==2.15
  * Fix copyrights.
  * Add downloading source files in tutorial folder.  (example_effdet_keras_mixed_precision_ptq.ipynb)
* Add explanation how to downgrade python3.12 to python3.11 in Colab environment for tutorials and README.md.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb
Comment (yt0705) [inline tutorials/notebooks/imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb:1]: > Extract ImageNet validation dataset using using 'prepare_imagenet.sh' script

Please correct the typos at this time.(using using)

Comment (ueno-hiroshi002) [reply tutorials/notebooks/imx500_notebooks/keras/example_keras_mobilenetv2_for_imx500.ipynb:1]: fix at b8cf4d3
